templates/configmaps_st2-conf.yaml

```diff
@@ -48,4 +47,10 @@ data:
   # User-defined st2 config with custom settings applied on top of everything else.
   # The order of merging: st2.conf < st2.docker.conf < st2.user.conf
   st2.user.conf: |
+  {{- if .Values.st2.rbac.enabled }}
+    [rbac]
+    sync_remote_groups = True
```

Comment:
Let's leave the `sync_remote_groups = True` up to users instead of injecting it by default
https://docs.stackstorm.com/latest/rbac.html#automatically-granting-roles-based-on-ldap-group-membership
as that'll work only when LDAP auth is enabled beforehand.